vispy/visuals/__init__.py

```diff
@@ -31,6 +31,7 @@
 from .regular_polygon import RegularPolygonVisual  # noqa
 from .scrolling_lines import ScrollingLinesVisual  # noqa
 from .spectrogram import SpectrogramVisual  # noqa
+from .sphere import SphereVisual # noqa
```

Comment:
there should be two spaces before the `# noqa`. So, the line should look like this:

```python
from .sphere import SphereVisual  # noqa
```